Walkthrough Guide/12 Anayltics/README.md

```diff
@@ -20,16 +22,16 @@ App Center Analytics will help you understand more about your app users and thei
 ## Push 
 Use App Center to easily send targeted and personalised push notifications to any mobile platform from any cloud or on-premises backend.
 
-Push notifications is vital for consumer apps and a key compontent in increasing app engagement and usage. For enterprise apps, it can also be used to help communicate up-to-date business information. It is the best app-to-user communication because it is energy-efficient for mobile devices, flexible for the notifications senders, and available while corresponding apps are not active.
+Push notifications is vital for consumer apps and a key compontent in increasing app engagement and usage. For enterprise apps, it can also be used to help communicate up-to-date business information. It is the best for app-to-user communication because it is energy-efficient for mobile devices, flexible for the notifications senders, and available while corresponding apps are not active.
 
 ### How Push Notifications Works
-Push notifications are delivered through platform-specific infrastructures called Platform Notification Systems (PNSes). They offer barebone push functionalities to delivery message to a device with a provided handle, and have no common interface. To send a notification to all customers across iOS and Android, we have to  work with APNS (Apple Push Notification Service) and FCM (Firebase Cloud Messaging).
+Push notifications are delivered through platform-specific infrastructures called Platform Notification Systems (PNSes). They offer barebone push functionalities to deliver a message to a device with a provided handle, and have no common interface. To send a notification to all customers across iOS and Android, we have to  work with APNS (Apple Push Notification Service) and FCM (Firebase Cloud Messaging).
 
 At a high level, here is how push works:
 
-1. The client app decides it wants to receive pushes hence contacts the corresponding PNS to retrieve its unique and temporary push handle. The handle type depends on the system (e.g. WNS has URIs while APNS has tokens).
-2. The client app stores this handle in the app back-end or provider.
-3. To send a push notification, the app back-end contacts the PNS using the handle to target a specific client app.
+1. The client app decides it wants to receive pushes hence it contacts the corresponding PNS to retrieve its unique and temporary push handle. The handle type depends on the system (e.g. WNS has URIs while APNS has tokens).
+2. The client app stores this handle in the app backend or provider.
```

Comment:
Same as above. Even though it was not in that notion before, please use `1.` for every single list item. Markdown will create the list automatically then.